src/elastio-snap.c

```diff
@@ -891,13 +891,18 @@ static void bio_free_pages(struct bio *bio){
 #define BIO_MAX_PAGES BIO_MAX_VECS
 #endif
 
+#define MAX_INODE_DEBUG_LIST 16
+
 //global module parameters
 static int elastio_snap_may_hook_syscalls = 0;
 static unsigned long elastio_snap_cow_ext_buf_size = sizeof(struct fiemap_extent) * 1024;
 static unsigned long elastio_snap_cow_max_memory_default = (300 * 1024 * 1024);
 static unsigned int elastio_snap_cow_fallocate_percentage_default = 10;
 static unsigned int elastio_snap_max_snap_devices = ELASTIO_SNAP_DEFAULT_SNAP_DEVICES;
 static int elastio_snap_debug = 0;
+static int elastio_snap_debug_data_dump = 1;
+int inode_debug_list[MAX_INODE_DEBUG_LIST];
+int inodes_count;
```

Comment:
Need initialize:
`int inodes_count = 0;`

Reply:
Why not! It's a global variable, but let's keep it pretty anyway.